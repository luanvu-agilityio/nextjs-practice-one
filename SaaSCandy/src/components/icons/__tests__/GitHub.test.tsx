import { render } from '@testing-library/react';
import GitHubIcon from '../GitHub';

describe('AddressIcon', () => {
  it('matches snapshot', () => {
    const { container } = render(<GitHubIcon />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
