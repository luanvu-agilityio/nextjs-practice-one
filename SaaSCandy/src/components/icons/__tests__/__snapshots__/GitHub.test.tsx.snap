// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`AddressIcon > matches snapshot 1`] = `
<svg
  fill="none"
  height="25"
  viewBox="0 0 25 25"
  width="25"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clip-path="url(#clip0_47_4437)"
  >
    <path
      d="M16.2917 22.4715V18.6015C16.3293 18.1247 16.2648 17.6453 16.1028 17.1953C15.9407 16.7453 15.6847 16.3349 15.3517 15.9915C18.4917 15.6415 21.7917 14.4515 21.7917 8.9915C21.7915 7.59532 21.2544 6.25269 20.2917 5.2415C20.7476 4.02 20.7154 2.66984 20.2017 1.4715C20.2017 1.4715 19.0217 1.1215 16.2917 2.9515C13.9998 2.33032 11.5837 2.33032 9.29175 2.9515C6.56175 1.1215 5.38175 1.4715 5.38175 1.4715C4.86812 2.66984 4.83589 4.02 5.29175 5.2415C4.32188 6.26019 3.78427 7.61496 3.79175 9.0215C3.79175 14.4415 7.09175 15.6315 10.2317 16.0215C9.90275 16.3615 9.64901 16.7669 9.48706 17.2114C9.3251 17.6559 9.25856 18.1296 9.29175 18.6015V22.4715M9.29175 19.4715C4.29175 20.9715 4.29175 16.9715 2.29175 16.4715L9.29175 19.4715Z"
      stroke="white"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </g>
  <defs>
    <clippath
      id="clip0_47_4437"
    >
      <rect
        fill="white"
        height="24"
        transform="translate(0.291748 0.471436)"
        width="24"
      />
    </clippath>
  </defs>
</svg>
`;

exports[`AddressIcon matches snapshot 1`] = `
<svg
  fill="none"
  height="25"
  viewBox="0 0 25 25"
  width="25"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clip-path="url(#clip0_47_4437)"
  >
    <path
      d="M16.2917 22.4715V18.6015C16.3293 18.1247 16.2648 17.6453 16.1028 17.1953C15.9407 16.7453 15.6847 16.3349 15.3517 15.9915C18.4917 15.6415 21.7917 14.4515 21.7917 8.9915C21.7915 7.59532 21.2544 6.25269 20.2917 5.2415C20.7476 4.02 20.7154 2.66984 20.2017 1.4715C20.2017 1.4715 19.0217 1.1215 16.2917 2.9515C13.9998 2.33032 11.5837 2.33032 9.29175 2.9515C6.56175 1.1215 5.38175 1.4715 5.38175 1.4715C4.86812 2.66984 4.83589 4.02 5.29175 5.2415C4.32188 6.26019 3.78427 7.61496 3.79175 9.0215C3.79175 14.4415 7.09175 15.6315 10.2317 16.0215C9.90275 16.3615 9.64901 16.7669 9.48706 17.2114C9.3251 17.6559 9.25856 18.1296 9.29175 18.6015V22.4715M9.29175 19.4715C4.29175 20.9715 4.29175 16.9715 2.29175 16.4715L9.29175 19.4715Z"
      stroke="white"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </g>
  <defs>
    <clippath
      id="clip0_47_4437"
    >
      <rect
        fill="white"
        height="24"
        transform="translate(0.291748 0.471436)"
        width="24"
      />
    </clippath>
  </defs>
</svg>
`;
